src/test/java/com/radixdlt/crypto/HashTest.java

```diff
@@ -94,23 +99,23 @@ public void from_engine___testCompare() {
 	public void from_engine___testHashValues256() {
 		assertArrayEquals(
 			Bytes.fromHexString("7ef0ca626bbb058dd443bb78e33b888bdec8295c96e51f5545f96370870c10b9"),
-			hash256(Longs.toByteArray(0L))
+				Hash.hash256(Longs.toByteArray(0L))
 		);
 		assertArrayEquals(
 			Bytes.fromHexString("3ae5c198d17634e79059c2cd735491553d22c4e09d1d9fea3ecf214565df2284"),
-			hash256(Longs.toByteArray(1L))
+				Hash.hash256(Longs.toByteArray(1L))
 		);
 		assertArrayEquals(
 			Bytes.fromHexString("d03524a98786b780bd640979dc49316702799f46bb8029e29aaf3e7d3b8b7c3e"),
-			hash256(Longs.toByteArray(10L))
+				Hash.hash256(Longs.toByteArray(10L))
 		);
 		assertArrayEquals(
 			Bytes.fromHexString("43c1dd54b91ef646f74dc83f2238292bc3340e34cfdb2057aba0193a662409c5"),
-			hash256(Longs.toByteArray(100L))
+				Hash.hash256(Longs.toByteArray(100L))
 		);
 		assertArrayEquals(
 			Bytes.fromHexString("4bba9cc5d36a21b9d09cd16bbd83d0472f9b95afd2d7aa1621bb6fa580c99bfc"),
-			hash256(Longs.toByteArray(1_000L))
+				Hash.hash256(Longs.toByteArray(1_000L))
```

Comment:
Weird spacing going on in this file

Reply:
yeah IntelliJ is freaking out sometimes... hmm...  will investigate.